shared/reducers/team-building.tsx

```diff
@@ -62,8 +62,8 @@ export const editTeambuildingDraft = (
       draftState.userRecs = action.payload.users
     },
     [TeamBuildingGen.search]: (draftState, action) => {
-      const {query, service, limit = draftState.searchLimit} = action.payload
```

Comment:
this actually fixes a long standing bug where if you search, then scroll down until it stops, you can't search anymore